### DISCUSSION TITLE
Banning cell types

### DISCUSSION THREAD

**Discussion by TurqW** [MaxRobinsonTheGreat/EvolutionSimulatorV2#41] [Ideas and Suggestions]
I'll admit that it probably doesn't make sense to ban mouth or producer cells, but interesting experiments could be done by selectively banning the other types in the simulation controls. It might even be interesting to be able to adjust the relative probabilities of each cell type (when mutating).

**Comment by Coolboy-10** [comment]
That's a good idea!
